docs/source/hardware/index.md

```diff
@@ -10,35 +10,37 @@ partitions
 ```
 Understanding the hardware architecture of the {term}`cluster` is key to maximizing the performance of your computations on our {term}`High-Performance Computing (HPC)`. This section introduces you to the hardware components, including an overall overview and detailed information about the various {term}`partitions <partition>`.
```

Comment:
including an overall overview and detailed information about the various. Remove "Overall" from the sentence.